src/engine/Input/Pointer.ts

```diff
@@ -66,35 +67,86 @@ const ScrollWheelNormalizationFactor = -1 / 40;
  */
 export class PointerEvent extends GameEvent<any> {
 
+   /** Deprecated. Use [[PointerEvent]].worldPos.x instead. */
```

Comment:
I believe we can use @eonarheim's `@Obsolete()` decorator for all these

Reply:
Please add this decorator code to deprecated things, it'll show warnings in the console if anything uses it.
```typescript
import { obsolete } from '../Util/Decorators';
...
@obsolete({message: 'PointerEvent.x will be removed in the 0.17 release', alternateMethod: 'PointerEvent.worldPos.x'})
public get x(): number {
```

In the JSDOC comment please change `Deprecated` to `@deprecated will be removed in the 0.17 release, please use PointerEvent.worldPos.x instead.` our documentation generation should pick this up and deploy the change when merged.

Reply:
I've corrected my comment above, the JSDOC should use `@obsolete` for doc generation